### PR TITLE
fix group list not showing proper number of users in Cypress (Should fix #706)

### LINKED ISSUE #706: Frontend tests sometimes fail: Expected to find element: '[data-test="signupBtn"]'
### Problem

Sometimes frontend tests will fail:

```
  1) SignUp, Profile and Login sign up new user1:
     CypressError: Timed out retrying: Expected to find element: '[data-test="signupBtn"]', but never found it.


  [... snip ...]

  (Results)

  ┌────────────────────────────────────────┐
  │ Tests:        9                        │
  │ Passing:      1                        │
  │ Failing:      1                        │
  │ Pending:      0                        │
  │ Skipped:      7                        │
  │ Screenshots:  1                        │
  │ Video:        false                    │
  │ Duration:     6 seconds                │
  │ Spec Ran:     signup-and-login.spec.js │
  └────────────────────────────────────────┘

  (Screenshots)

  - group-income-simple/test/cypress/screenshots/signup-and-login.spec.js/SignUp, Profile and Login -- sign up new user1 (failed).png (1280x720)
```

The screenshot it saves is:

SignUp, Profile and Login -- sign up new user1 (failed)

### Solution

Fix the test :)

### PR DESCRIPTION
**Pull request by taoeffect**
Hopefully this fixes our cypress tests!

### CODE REVIEW

File: frontend/model/state.js [hunk 1]
```diff
@@ -6,21 +6,21 @@
...
 import { ErrorDBBadPreviousHEAD, ErrorDBConnection } from '~/shared/domains/gi/db.js'
-import { LOGIN, LOGOUT, EVENT_HANDLED, CONTRACTS_MODIFIED } from '@utils/events.js'
 import Colors from './colors.js'
 import { TypeValidatorError } from '@utils/flowTyper.js'
 import { GIErrorUnrecoverable, GIErrorIgnoreAndBanIfGroup, GIErrorDropAndReprocess } from './errors.js'
 import { STATUS_OPEN, PROPOSAL_REMOVE_MEMBER } from './contracts/voting/proposals.js'
 import { VOTE_FOR } from '@model/contracts/voting/rules.js'
 import { actionWhitelisted, CONTRACT_REGEX } from '@model/contracts/Contract.js'
 import { currentMonthTimestamp } from '@utils/time.js'
+import * as _ from '@utils/giLodash.js'
```
Comment (pieer) [inline frontend/model/state.js:22]: Do we need every function to lodash? Maybe we should add a comment to import only the necessary functions (sorry for out of context)

**Review by pieer — DISMISSED** [review]

File: frontend/model/state.js (hunk 1 above)
Comment (taoeffect) [reply frontend/model/state.js:22]: We use a large number of functions in this file so it's OK to include `_`. The project uses everything defined in `giLodash.js` so tree shaking isn't a problem here

**Review by SebinSong — DISMISSED** [review]
Great work.

**Review by sandrina-p — APPROVED** [review]

**Comment by taoeffect** [comment]
**NOTE: this is an exceptional merge that violates Rule 7 of our contributions guidelines because it's during a hackathon and I have approval from @sandrina-p to merge it, and @sandrina-p cannot merge it herself because the build is failing, but it's an error unrelated to this PR.**

**Pull request merged**